wrapping/dockcross-manylinux-build-module-wheels-opencl.sh

```diff
@@ -20,6 +23,12 @@ chmod u+x /tmp/dockcross-manylinux-x64
 
 script_dir=$(cd $(dirname $0) || exit 1; pwd)
 
+mkdir -p $(pwd)/tools
+chmod 777 $(pwd)/tools
```

Comment:
Perhaps no-one else could be logged in to interfere, but if `755` works then I'd use it.

Reply:
For now I will keep this as-is for consistency with the source script in https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-build-module-wheels.sh#L40 . I agree that the security risk is low as the runner has a limited lifetime. Open to testing and discussion in ITKPythonPackage.